src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/mgmt_vmss/nested_templates/load_balancer_new.json

```diff
@@ -0,0 +1,348 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "_artifactsLocation": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "URL of the template to deploy."
+      }
+    },
+    "backendPoolName": {
+      "type": "string",
+      "defaultValue": "[concat(parameters('loadBalancerName'), 'bepool')]",
+      "metadata": {
+        "description": "Name of load balancer backend pool."
+      }
+    },
+    "natPoolName": {
+      "type": "string"
+    },
```

Comment:
we generally haven't kept the child templates sorted, but we might as well start :)

---
In reply to: [76839539](https://github.com/Azure/azure-cli/pull/774#discussion_r76839539) [](ancestors = 76839539)